### PR TITLE
Adding pyinstaller example.

### PR DESCRIPTION
**Pull request by martinzak-zaber**

### CODE REVIEW

File: src/util_pyinstaller/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+```ps1
+.\setup_venv.bat
+.\build.bat
+```
+
+You'll find the distributable executable `main.exe` in the `dist` directory.
+
+## Running the Script
+
+To run the script:
```
Comment (andrewbrucelau-zaber) [inline src/util_pyinstaller/README.md:38]: Suggestion: move this section above "To build the executable", so people don't confuse this section as executing the already compiled executable.  Also, maybe add a section that they should change the COM port in the script before running.

File: src/util_pyinstaller/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## Building the executable
+
+To build the executable:
+
+```ps1
+.\setup_venv.bat
+.\build.bat
+```
+
+You'll find the distributable executable `main.exe` in the `dist` directory.
```
Comment (andrewbrucelau-zaber) [inline src/util_pyinstaller/README.md:34]: Maybe put in command to execute `main.exe`

File: src/util_pyinstaller/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Packaging ZML with PyInstaller
+
+*By Martin Zak*
+
+This example demonstrates how to package a Python script that uses Zaber Motion Library with PyInstaller.
+The script script itself in `main.py` is a simple motion example.
```
Comment (andrewbrucelau-zaber) [inline src/util_pyinstaller/README.md:6]: script script...

File: src/util_pyinstaller/pyvenv.cfg
Comment (andrewbrucelau-zaber) [inline src/util_pyinstaller/pyvenv.cfg:1]: This file accidentally included?

**Review by andrewbrucelau-zaber — APPROVED** [review]
Looks great!  Just a few minor things.

File: src/util_pyinstaller/README.md (hunk 1 above)
Comment (martinzak-zaber) [reply src/util_pyinstaller/README.md]: Done

File: src/util_pyinstaller/README.md (hunk 3 above)
Comment (martinzak-zaber) [reply src/util_pyinstaller/README.md]: Done

File: src/util_pyinstaller/pyvenv.cfg
Comment (martinzak-zaber) [reply src/util_pyinstaller/pyvenv.cfg:1]: Removed.

**Pull request merged**